dallinger/experiment.py

```diff
@@ -945,12 +952,38 @@ def load():
         except ImportError:
             from dallinger_experiment import dallinger_experiment as experiment
 
-        classes = inspect.getmembers(experiment, inspect.isclass)
-        for name, c in classes:
-            if "Experiment" in c.__bases__[0].__name__:
-                return c
+        classes = inspect.getmembers(experiment, is_experiment_class)
+
+        preferred_class = os.environ.get("EXPERIMENT_CLASS_NAME", None)
+        if preferred_class is not None:
+            try:
+                return dict(classes)[preferred_class]
+            except KeyError:
+                raise ImportError(
+                    "No experiment named {} was found".format(preferred_class)
+                )
+
+        if len(classes) > 1:
+            for name, c in classes:
+                if "Experiment" in c.__bases__[0].__name__:
```

Comment:
I can see that this is the same test that was being used before, but I don't think I understand what it's supposed to be checking. Right now it checks that the name of the first class inherited from contains the word `Experiment`.

Is that really what we want to check for? I don't understand why that wasn't just `issubclass` and with your updated `is_experiment_class` check that already does `issubclass` doesn't it just become unnecessary?

Reply:
Yep, that's exactly what happens. The old logic uses the last name (sorted alphabetically) which is a class whose immediate parent contains the string Experiment. It's terrible. I've included this because I don't want to break things for people with multiple classes, but that's why I've added the warning.